### ISSUE TITLE
Instructions keyboard shortcut doesn't work on windows

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#277186]
Discovered in https://github.com/microsoft/vscode/issues/269075 - have not been able to confirm myself

- Using the attach context picker, see keyboard shortcut for `Instructions`. 
- Try it, doesn't work. Isn't announced to screen reader users (that's good)

We should remove it visually
